### DISCUSSION TITLE
file:// scheme problem

### DISCUSSION THREAD

**Discussion by imichael66** [curl/curl#8508] [Q&A]
Hello,
Seems to be a problem with the "file://" scheme
after recompiling curl i have this error

$ ./curl  file:///D/fileA.txt
curl: (37) Couldn't open file /D/fileA.txt

I know the file exists D:\fileA.txt

Extra info:
-----------
(1) Here is my ./configure output:
configure: Configured to build curl/libcurl:

  Host setup:       x86_64-w64-mingw32
  Install prefix:   /mingw64
  Compiler:         gcc
   CFLAGS:          -Werror-implicit-function-declaration -O2 -Wno-system-headers -Wenum-conversion
   CPPFLAGS:        -isystem C:/msys64/mingw64/include -isystem ../openssl-3.0.1/include -isystem ../openssl-3.0.1/include/openssl -DOPENSSL_SUPPRESS_DEPRECATED
   LDFLAGS:         -LC:/msys64/mingw64/lib -L../openssl-3.0.1/lib
   LIBS:            -ladvapi32 -lcrypt32 -lssl -lcrypto -lgdi32 -lwldap32 -lzstd -lz -lws2_32

  curl version:     7.81.0
  SSL:              enabled (OpenSSL v3+)
  SSH:              no      (--with-{libssh,libssh2})
  zlib:             enabled
  brotli:           no      (--with-brotli)
  zstd:             enabled (libzstd)
  GSS-API:          no      (--with-gssapi)
  GSASL:            no      (libgsasl not found)
  TLS-SRP:          enabled
  resolver:         POSIX threaded
  IPv6:             enabled
  Unix sockets:     no      (--enable-unix-sockets)
  IDN:              no      (--with-{libidn2,winidn})
  Build libcurl:    Shared=yes, Static=yes
  Built-in manual:  enabled
  --libcurl option: enabled (--disable-libcurl-option)
  Verbose errors:   enabled (--disable-verbose)
  Code coverage:    disabled
  SSPI:             no      (--enable-sspi)
  ca cert bundle:   no
  ca cert path:     no
  ca fallback:      no
  LDAP:             enabled (winldap)
  LDAPS:            enabled
  RTSP:             enabled
  RTMP:             no      (--with-librtmp)
  PSL:              no      (libpsl not found)
  Alt-svc:          enabled (--disable-alt-svc)
  HSTS:             enabled (--disable-hsts)
  HTTP1:            enabled (internal)
  HTTP2:            no      (--with-nghttp2, --with-hyper)
  HTTP3:            no      (--with-ngtcp2, --with-quiche)
  ECH:              no      (--enable-ech)
  Protocols:        DICT FILE FTP FTPS GOPHER GOPHERS HTTP HTTPS IMAP IMAPS LDAP LDAPS MQTT POP3 POP3S RTSP SMB SMBS SMTP SMTPS TELNET TFTP
  Features:         AsynchDNS HSTS HTTPS-proxy IPv6 Largefile NTLM SSL TLS-SRP alt-svc libz zstd

(2) im compiling for Windows 10 and Mingw64
MINGW64_NT-10.0-19042 SKMT-003 3.3.3-341.x86_64 2022-01-18 13:00 UTC x86_64 Msys

**Comment by jay** [comment]
> $ ./curl file:///D/fileA.txt
> curl: (37) Couldn't open file /D/fileA.txt

it's missing a colon, are you sure it's not a typo, file:///D:/fileA.txt

**Reply by imichael66** [reply]
Hi Jay.

the thing is the distribution curl gets the file
$ **curl**  file:///D/fileA.txt
---------------oTHIS IS A TEST FROM fileA.txto----------------

but the recompiled one get the error
$ **./curl**  file:///D/fileA.txt
curl: (37) Couldn't open file /D/fileA.txt

Best regards

**Comment by bagder** [comment]
This is a bug/issue, not a "discussion".

**Reply by bagder** [reply]
Continue at #8510

**Reply by imichael66** [reply]
Do ou want me to open a ticket/bug  on this topic then ?

**Reply by jay** [reply]
It looks like user error.
